srtcore/buffer.cpp

```diff
@@ -195,11 +195,11 @@ void CSndBuffer::addBuffer(const char* data, int len, SRT_MSGCTRL& w_mctrl)
     }
 
     const steady_clock::time_point time = steady_clock::now();
-    if (w_srctime == 0)
-    {
-        HLOGC(dlog.Debug, log << CONID() << "addBuffer: DEFAULT SRCTIME - overriding with current time.");
-        w_srctime = count_microseconds(time.time_since_epoch());
-    }
+    // if (w_srctime != 0)
+    // {
+    //     HLOGC(dlog.Debug, log << CONID() << "addBuffer: DEFAULT SRCTIME - overriding with current time.");
+    //     w_srctime = count_microseconds(time.time_since_epoch());
+    // }
```

Comment:
This is not needed because `time` is saved as origin anyway.

Reply:
But the original condition was to rewrite it in case when it's 0, that is, wasn't specified.

Reply:
Also, Origin Time is saved for a completely different purpose. Source time is saved so that it is set the same when retransmitting a packet in live mode.